packages/client/package.json

```diff
@@ -1,61 +1,70 @@
 {
-    "name": "@revertdotdev/revert-ui",
-    "version": "0.1.0",
-    "private": true,
-    "proxy": "http://localhost:4001",
-    "dependencies": {
-        "@clerk/clerk-react": "4.11.2",
-        "@emotion/react": "^11.11.1",
-        "@emotion/styled": "^11.11.0",
-        "@mui/icons-material": "^5.10.3",
-        "@mui/lab": "^5.0.0-alpha.135",
-        "@mui/material": "^5.10.3",
-        "@sentry/react": "^7.73.0",
-        "@testing-library/jest-dom": "^6.1.3",
-        "@testing-library/react": "^13.3.0",
-        "axios": "^1.4.0",
-        "react": "^18.2.0",
-        "react-dom": "^18.2.0",
-        "react-flow-renderer": "^10.3.16",
-        "react-github-btn": "^1.4.0",
-        "react-hot-toast": "^2.4.0",
-        "react-loader-spinner": "^5.3.4",
-        "react-router-dom": "^6.4.1",
-        "react-scripts": "5.0.1",
-        "styled-components": "^6.0.7",
-        "web-vitals": "^3.4.0"
-    },
-    "scripts": {
-        "start": "DANGEROUSLY_DISABLE_HOST_CHECK=true; react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test",
-        "eject": "react-scripts eject"
-    },
-    "eslintConfig": {
-        "extends": [
-            "react-app",
-            "react-app/jest"
-        ]
-    },
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not dead",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    },
-    "devDependencies": {
-        "@types/jest": "^29.5.4",
-        "@types/react": "^18.0.17",
-        "@types/react-dom": "^18.0.6",
-        "autoprefixer": "^10.4.16",
-        "postcss": "^8.4.16",
-        "tailwindcss": "^3.1.8",
-        "typescript": "^4.7.4"
-    }
+  "name": "@revertdotdev/revert-ui",
```

Comment:
I doubt we need to make any change to this file here. Please revert or let me know what's needed for Discord Integration in this file.